tests/test_main.py

```diff
@@ -1,6 +1,11 @@
+import mock
+import sys
+
 from validator.main import main
 
-def test_main():
+
+@mock.patch.object(sys, 'stderr')
```

Comment:
Essentially, this just tests that `main` runs up until the point that `parse_args` quits due to missing required arguments. The issue is that it prints a usage message directly to stderr, which shows up in the test output.

We really need proper tests for `main.py`. This test is just acting as a stopgap to make sure that the file loads and initializes correctly.